libs/blocks/marketo/marketo.js

```diff
@@ -20,32 +20,10 @@ const HIDDEN_FIELDS = 'hidden fields';
 const BASE_URL = 'base url';
 const FORM_ID = 'form id';
 const MUNCHKIN_ID = 'munchkin id';
-const ERROR_MESSAGE = 'error message';
-
-/* Marketo adds default styles that we want to remove */
-const cleanStyleSheets = (baseURL) => {
-  const { styleSheets } = document;
-
-  [...styleSheets].forEach((sheet) => {
-    if (sheet.href?.includes(baseURL)) {
-      sheet.disabled = true;
-    }
-  });
-};
-
-const cleanFormStyles = (form) => {
-  const formEl = form.getFormElem().get(0);
-
-  formEl?.querySelectorAll('style').forEach((e) => { e.remove(); });
-  formEl?.parentElement?.querySelectorAll('*[style]').forEach((e) => e.removeAttribute('style'));
-};
```

Comment:
A little surprised you've deleted this. Are we leveraging some of the styles sent with the forms? Just curious.